code/game/objects/effects/decals/contraband.dm

```diff
@@ -362,6 +365,33 @@
 	desc = "The Griffin commands you to be the worst you can be. Will you?"
 	icon_state = "poster33"
 
+/obj/structure/sign/poster/contraband/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/toy/crayon/red/syndicate))
+		to_chat(usr, "<span class = 'notice'>There is no reason to change the message of this poster.</span>")
+		return
```

Comment:
Shouldnt there be an ..() here? otherwise you overwrite the proc and prevent other attackbys

Reply:
oh yes i missed that, Fixing this and the other one in the next commit